docs/anchoring-assets/anchor-platform/sep6/integration.mdx

```diff
@@ -0,0 +1,1103 @@
+```
+
+</CodeExample>
+
+After successful processing of the payment on a custody service, the Anchor Platform will automatically make the `notify_onchain_funds_sent` JSON-RPC request and the status of the transaction will be changed to `completed`.
+
+:::caution
+
+A user account may not be ready to receive funds. You can check that the account has established a [trustline](/docs/glossary#trustline). Otherwise, you can set the status of the transaction to the `pending_trust` to indicate that the anchor is waiting for the user to establish the trustline.
+
+If custody integration is enabled, the Anchor Platform will do this validation for you automatically.
+
+:::
+
+### Pending Trust
+
+This status has to be set if a payment requires an asset trustline that wasn't configured by the user. There are two ways of how the transaction may be moved to the `pending_trust` status. The first one is processing of a payment via custody service in case it detected that the trustline isn't configured. The second one is when the business itself detects that the trustline is missing and wants to notify the user that it has to be configured. To move the transaction to the `pending_trust` status, make the following JSON-RPC request.
+
+<CodeExample>
+
+```json
+// request-trust.json
+[
+  {
+    "id": 1,
+    "jsonrpc": "2.0",
+    "method": "request_trust",
+    "params": {
+      "transaction_id": "<transaction_id>",
+      "message": "Asset trustine not configured"
+    }
+  }
+]
+```
+
+</CodeExample>
+
+To execute this, you need to run:
+
+<CodeExample>
+
+```bash
+./call-json-rpc.sh request-trust.json
+```
+
+</CodeExample>
+
+:::info
+
+Payment via custody service periodically checks if the trustline was configured. If it was, it will automatically send a payment to a custody service and change the status of the transaction to `pending_stellar`.
+
+:::
+
+### Trust Set
+
+This status has to be set if the business has detected that the trustline was or wasn't configured by user.
+
+<CodeExample>
+
+```json
+// trust-set.json
+[
+  {
+    "id": 1,
+    "jsonrpc": "2.0",
+    "method": "notify_trust_set",
+    "params": {
+      "transaction_id": "<transaction_id>",
+      "message": "Asset trustine set",
+      "success": "true"
+    }
+  }
+]
+```
+
+</CodeExample>
+
+- `success` flag which defines if trustline was or wasn't configured by user
+
+To execute this, you need to run:
+
+<CodeExample>
+
+```bash
+./call-json-rpc.sh trust-set.json
+```
+
+</CodeExample>
+
+:::info
+
+Depending on the `success` flag, the status of the transaction will be changed to `pending_stellar` if the trustline was set, or to `pending_anchor` if it wasn't.
+
+:::
+
+### Refund Sent
+
+Sometimes, funds need to be sent back to the user (refund). You can refund the whole sum (full refund) or do a set of partial refunds. Also, if user sent more money than expected, you can refund a part of the sum back to the user and send the rest as onchain funds.
```

Comment:
I think it's good to mention that business needs to send stellar transaction back to user account (back to the `source_account`), using `refund_memo` and `refund_memo_type` (if present)
probably should also clarify this part in SEP-24 guide